test/README.md

```diff
@@ -53,48 +53,48 @@ All FIT tests can be run from the wrapper 'run_tests.py':
 
         Command Help
 
         optional arguments:
-          -h, --help          show this help message and exit
-          -test TEST          test to execute, default: tests/
-          -config CONFIG      config file location, default: config
-          -group GROUP        test group to execute: 'smoke', 'regression',
-                              'extended', default: 'all'
-          -stack STACK        stack label (test bed), overrides -ora
-          -ora ORA            OnRack/RackHD appliance IP address or hostname, default:
-                              localhost
-          -version VERSION    OnRack package install version, example:onrack-
-                              release-0.3.0, default: onrack-devel
-          -template TEMPLATE  path or URL link to OVA template or OnRack OVA
-          -xunit              generates xUnit XML report files
-          -numvms NUMVMS      number of virtual machines for deployment on specified
-                              stack
-          -list               generates test list only
-          -sku SKU            node SKU name, example: Quanta-T41, default=all
-          -obmmac OBMMAC      node OBM MAC address, example:00:1e:67:b1:d5:64
-          -nodeid NODEID      node identifier string of a discovered node, example:
-                              56ddcf9a8eff16614e79ec74
-          -http               forces the tests to utilize the http API protocol
-          -https              forces the tests to utilize the https API protocol
-          -port PORT          API port number override, default from
-                              install_config.json
-          -v V                Verbosity level of console and log output (see -nose-
-                              help for more options), Built Ins: 0: Minimal logging,
-                              1: Display ERROR and CRITICAL to console and to files,
-                              3: Display INFO to console and to files, 4: (default)
-                              Display INFO to console, and DEBUG to files, 5: Display
-                              infra.run and test.run DEBUG to both, 6: Add display of
-                              test.data (rest calls and status) DEBUG to both, 7: Add
-                              display of infra.data (ipmi, ssh) DEBUG to both, 9:
-                              Display infra.* and test.* at DEBUG_9 (max output)
-          -nose-help          display help from underlying nosetests command,
-                              including additional log options
+          -h, --help            show this help message and exit
+          -test TEST            test to execute, default: tests/
+          -config CONFIG        config file location, default: config
+          -group GROUP          test group to execute: 'smoke', 'regression',
+                                'extended', default: 'all'
+          -stack STACK          stack label (test bed)
+          -rackhd_host RACKHD_HOST
```

Comment:
@stuart-stanley The agreed on name is -rackhd_host because that is the legacy representation in CIT testing and was specified in the story. We went back-and-forth about this and I don't want to change it now.